framework/tlvf/src/src/CmduMessageRx.cpp

```diff
@@ -52,6 +52,10 @@ bool CmduMessageRx::parse(uint8_t *buff, size_t buff_len, bool swap_needed, bool
     if (!parse_tlvs)
         return true;
 
+    /**
+     * MMZ
+     * Why using a list of parsers? It seems like just one parser should suffice
```

Comment:
I had the same feedback but the PR got merged regardless :-)

Should be fixed soon by @CoralMalachi and @Galvien as part of #318 #471 #160